lib/Serialization/ModuleFormat.h

```diff
@@ -2342,6 +2342,11 @@ namespace decls_block {
     TypeIDField                       // result type
   >;
 
+  using WarnDeclAttrLayout = BCRecordLayout<
+    Warn_DECL_ATTR,
+    BCFixed<1> // implicit flag
+  >;
```

Comment:
This is a mostly safe addition but I'd bump up `SWIFTMODULE_VERSION_MINOR` if you happen to make further changes to this PR, especially if we want to cherry-pick this to other branches.